### DISCUSSION TITLE
multiple plots mount points

### DISCUSSION THREAD

**Discussion by panayotovip** [guydavis/machinaris#243] [Q&A] [answered]
how to add multiple plot mount points ?
in docker primary mount point is
`-v /srv/plots1:/plots:rw`
add secondary like this
`-v /srv/plots2:/plots2:rw`
then how to config to use second folder too ?
(in container is /plots and /plots2)
when copy plots in host /srv/plots2 in container /plots2 is shown
but do not show that plots in summary of machinaris

p.s.: sorry for bad English, mostly use google translate.

**Comment by istarion** [comment]
@panayotovip , you can configure additional directories in farmer and plotter config.

Farmer:
```
harvester:
...
  plot_directories:
  - /plots
  - /plots2
```

Plotter:
```
directories:
...
  dst:
    - /plots
    - /plots2
```

**Reply by panayotovip** [reply]
That worked !
In summary now show all available plots.
But when go to Farming in list there show plots from /plots in container.
There is no plots from /plots2.
Interesting is that when i do check plots, all plots are shown and checks correctly.

p.s.: sorry for bad English, mostly use google translate.

**Comment by guydavis** [comment, marked as answer]
Hello! Please see this [FAQ answer](https://github.com/guydavis/machinaris/wiki/FAQ#can-i-use-two-or-more-final-destination-directories-to-hold-my-plots).  The correct Docker run/compose is generated automatically if you complete the form at [https://www.machinaris.app/](https://www.machinaris.app/). Hope this helps.

**Reply by panayotovip** [reply]
it works. thanks.
